index.js

```diff
@@ -0,0 +1,107 @@
+const express = require('express');
+const helmet = require('helmet');
+const knex = require('knex');
+const bcrypt = require('bcryptjs');
+const session = require('express-session');
+const KnexSessionStore = require('connect-session-knex')(session);
+
+
+const knexConfig = require('./knexfile.js');
+const db = knex(knexConfig.development);
+
+const server = express();
+
+const sessionConfig = {
+    secret: 'Extremelysecretthinghere',
+    name: 'brad',
+    httpOnly: true,
+    saveUninitialized: false,
+    cookie: {
+        secure: false,
+        maxAge: 1000 * 60 * 10
```

Comment:
It seems setting 10 minutes for cookie max age can sometimes cause issues as i was unable to login. Needed to alter the max-age to at least 1 hour. Just wanted to give headsup that it can be an issue if You are not able to view protected route and are already logged in.